dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.17.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib flag-o-matic
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="mirror://mariadb/connector-odbc-${PV}/${P}-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.1"
+KEYWORDS="amd64 x86"
+IUSE="ssl"
+
+S="${S}-src"
```

Comment:
These days we put this directly below SRC_URI because its path is directly related to the tarball.